src/components/App/App.css

```diff
@@ -0,0 +1,4 @@
+body {
+    margin: 0;
+    padding: 0;
+}
```

Comment:
What's the reason this is required...?  Platform should do it for you in the CSS Reset?

Reply:
Yeah you're right, removed in 98dac0f